openquake/risklib/scientific.py

```diff
@@ -950,78 +953,15 @@ def __call__(self, assets, ratios_by_aid, ses_ratio):
                 continue
             counts = numpy.array([(loss_ratios >= ratio).sum(axis=0)
                                   for ratio in self.ratios])
+            poes = build_poes(counts, 1. / ses_ratio)
```

Comment:
Please add a comment about the fact this is never going to produce a division by zero